variables-urls.tf

```diff
@@ -0,0 +1,70 @@
+variable "ROOT_DOMAIN" {
+  type        = string
+  description = "Full URL the instance will be published on. Should not have a trailing slash."
+}
+
+locals {
+  address_db_service_name      = "address-db"
+  catalog_db_service_name      = "catalog-db"
+  discount_db_service_name     = "discount-db"
+  inventory_db_service_name    = "inventory-db"
+  invoice_db_service_name      = "invoice-db"
+  media_db_service_name        = "media-db"
+  notification_db_service_name = "notification-db"
+  order_db_service_name        = "order-db"
+  payment_db_service_name      = "payment-db"
+  review_db_service_name       = "review-db"
+  return_db_service_name       = "return-db"
+  shipment_db_service_name     = "shipment-db"
+  shoppingcart_db_service_name = "shoppingcart-db"
+  tax_db_service_name          = "tax-db"
+  user_db_service_name         = "user-db"
+  wishlist_db_service_name     = "wishlist-db"
+
+  otel_collector_service_name = "otel-collector"
+
+  db_port = 5432
+  otel_collector_port = 4317
+}
+locals {
+  // The Postgresql HA Helm chart always appends '-postgresql', so we would need to add it to the URL too, if we switched to it
```

Comment:
do we ever plan to switch to the HA chart?

Reply:
I originally tried to use the HA chart, but it made too many problems.
However, there is a second reason why we need it: If we ever decide to switch from a deployment to a statefulset, we will need this variables again.
As such, I simply left the already written code in as it does not decrease maintainability and does not affect performance.